### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by delete1234** [Hellowlol/ha-tide#8]
This addon is not working abymore. Would love to see it fixed :)

**Comment by Hellowlol** [comment]
Mind posting a debug log?

**Comment by delete1234** [comment]
_Position outside area
19:56:11 – (WARNING) Tide (custom integration) - message first occurred at 3 September 2022 at 13:36:01 and shows up 472 times_

Checked and doublechecked + removed and added integration several times. Found another Tide Intergration which is working :)

**Comment by Hellowlol** [comment]
This is just for Norway. Did you set coordinates for another country?

**Comment by delete1234** [comment]
Yes, I'm in Norway and it was was working perfectly until some months ago. NWM, if this is only me having a problem, please don't bother yourself with this. Anyway, nice work. My LAT / LONG is set correctly
